src/contracts/Limitation/Type.php

```diff
@@ -143,7 +143,7 @@ public function getCriterion(APILimitationValue $value, APIUserReference $curren
     /**
      * Returns info on valid $limitationValues.
      *
-     * @return array<string, string>|Traversable|int In case of array, a hash with key as valid limitations value and value as human-readable name
+     * @return array<string, string>|int In case of array, a hash with key as valid limitations value and value as human-readable name
```

Comment:
Original error:
`Method Ibexa\Contracts\Core\Limitation\Type::valueSchema() has invalid return type Ibexa\Contracts\Core\Limitation\Traversable.`

To fix with the following brings a new error.
```suggestion
     * @return array<string, string>|\Traversable|int In case of array, a hash with key as valid limitations value and value as human-readable name
```
`Method Ibexa\Contracts\Core\Limitation\Type::valueSchema() return type has no value type specified in iterable type Traversable. See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type`

Reply:
(In fact, I have this in my PR #463 CI failure but I don't see how it could be related to my changes…)

Reply:
@adriendupuis `iterable` === `array|Traversable`.

So in this case:
```suggestion
     * @return iterable<string, string>|int In case of array, a hash with key as valid limitations value and value as human-readable name
```

Reply:
@Steveb-p
Thank you for this suggestion. I'm keeping it for a latter time.

`Traversable` was in fact added by me while trying to enhance this PHPDoc block according to its description.
I reverted my change. The original:
```php
    /**
     * Returns info on valid $limitationValues.
     *
     * @return mixed[]|int In case of array, a hash with key as valid limitations value and value as human readable name
     *                     of that option, in case of int on of VALUE_SCHEMA_* constants.
     *                     Note: The hash might be an instance of Traversable, and not a native php array.
     */
    public function valueSchema();
```

The actual rendering in https://doc.ibexa.co/en/latest/api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Limitation-Type.html#method_valueSchema is the signature `public valueSchema() : array<string|int, mixed>|int` and below. I don't know how to enhance that yet from #463, it needs study, and I will keep it for another time, another PR.